commandChanVim.py

```diff
@@ -123,26 +130,13 @@ def handleKey(self, key):
         if self.mode is MODE.NORMAL:
             rows, cols = urwid.raw_display.Screen().get_cols_rows()
             DEBUG(key)
-            if key == 'h':
-                self.body.keypress((rows,cols), 'left')
-            if key == 'j':
-                self.body.keypress((rows,cols), 'down')
-            if key == 'k':
-                self.body.keypress((rows,cols), 'up')
-            if key == 'l':
-                self.body.keypress((rows,cols), 'right')
-
-            if key == 'H':
-                self.body.keypress((rows,cols), 'left')
-            if key == 'J':
-                self.body.keypress((rows,cols), 'down')
-            if key == 'K':
-                self.body.keypress((rows,cols), 'up')
-            if key == 'L':
-                self.body.keypress((rows,cols), 'right')
-
+            if key.isalpha():
+                key = key.lower()
+            
             if key == 'r':
-                pass
+                return
+
+            self.body.keypress((rows, cols), urwidView.KEYMAP[key])
```

Comment:
A KeyError is thrown if a user presses a key not in the map. Can you update to 

```python
try:
    self.body.keypress((rows, cols), urwidView.KEYMAP[key])
except:
    pass
```

or something along those lines?

Reply:
Sounds good

Reply:
Updated @ginglis13